.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Release
+on:
+ pull_request:
+    branches:
+      - main
```

Comment:
Change it to workflow dispatch, as we won't had a release on every merge

Reply:
remove package.lock.json

Reply:
> remove package.lock.json

but it may cause some unexpected issues in the future.
it's generally recommended to keep the package-lock.json file in your Node.js project and avoid deleting it unless there's a specific reason to do so.